controllers/storagecluster/storageclasses.go

```diff
@@ -443,11 +443,15 @@ func getTopologyConstrainedPools(initData *ocsv1.StorageCluster) string {
 
 	var topologyConstrainedPools []topologyConstrainedPool
 	for _, failureDomainValue := range initData.Status.FailureDomainValues {
+		failureDomain := initData.Status.FailureDomain
+		if failureDomain == "host" {
```

Comment:
@malayparida2000 please comment it out why we do below

Reply:
@humblec Done